.infrastructure/deployment.yml

```
@@ -29,7 +29,12 @@ spec:
             cpu: "150m"
         env:
         - name: PYTHONUNBUFFERED
-          value: "1"
+          valueFrom:
+            configMapKeyRef:
+              name: todo-config
+              key: PYTHONUNBUFFERED
```

```suggestion
          valueFrom:
            configMapKeyRef:
              name: todo-config
              key: PYTHONUNBUFFERED
        - name: SECRET_KEY
          valueFrom:
            secretKeyRef:
              name: todoapp-app-secrets
              key: SECRET_KEY
```
You need to add such line